client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -233,7 +233,7 @@ const SiteSetupList = ( {
 					const isCompleted = task.isCompleted;
 
 					return (
-						<>
+						<Fragment key={ task.id }>
```

Comment:
Hunh, didn't realize we could do that. https://reactjs.org/docs/fragments.html#keyed-fragments

Reply:
Today I learned that <></> was a documented practise! I thought it was hack!